### PR TITLE
Connect grant/revoke admin endpoints to buttons

### PR DESCRIPTION
**Pull request by juliasaveliff**
- When click "Remove Admin" or "Add Admin" on a user table, confirmation dialog appears, then toast displays outcome, and table updates
- Also fixed a bug coming from LOGGING delete actions that was messing with delete actions

### CODE REVIEW

File: rackcity/utils/log_utils.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def log_delete(user, element_type, element_name):
         element_name + ":",
         user.username,
         Action.DELETE.value,
-        element_type,
+        element_type.value,
```
Comment (juliasaveliff) [inline rackcity/utils/log_utils.py]: Bug fix from log (need the enum value which is a str)

File: src/components/elementView/elementTable.tsx [hunk 2]
```diff
@@ -64,7 +67,8 @@ interface ElementTableState {
   fields: Array<string>;
   isEditFormOpen: boolean;
   editFormValues: ElementObjectType;
-  isDeleteOpen: boolean;
+  openAlert: ElementTableOpenAlert;
```
Comment (juliasaveliff) [inline src/components/elementView/elementTable.tsx]: other option would have been to have 3 bools on state (isDeleteOpen, isGrantAdminOpen, isRevokeAdminOpen) to handle all alerts, which felt gross!

File: src/components/elementView/elementTable.tsx [hunk 3]
```diff
@@ -691,12 +760,32 @@ class ElementTable extends React.Component<
...
         >
           <p>Are you sure you want to delete?</p>
         </Alert>
+        <Alert
+          cancelButtonText="Cancel"
+          confirmButtonText="Confirm"
+          intent="danger"
+          isOpen={this.state.openAlert === ElementTableOpenAlert.GRANT_ADMIN}
+          onCancel={this.handleGrantAdminCancel}
+          onConfirm={this.handleGrantAdmin}
```
Comment (juliasaveliff) [inline src/components/elementView/elementTable.tsx]: this nugget right here is why I created the `selected_userid` field, the method `handleGrantAdmin` needs to know which user was selected when this button was clicked. if there is a better way to do this that i'm missing that doesn't required the selected user/row field on state, I can change this!

File: src/components/elementView/elementTable.tsx [hunk 4]
```diff
@@ -863,44 +952,44 @@ class ElementTable extends React.Component<
...
-                                onClick={(event: any) => {
-                                  this.handleDeleteButtonClick(item);
-                                  event.stopPropagation();
-                                }}
-                              />
-                              {this.props.isAdmin &&
-                              isAssetObject(item) &&
-                              item.rack.is_network_controlled ? (
+                            this.props.type !== ElementType.USER ? (
+                              <div className="inline-buttons">
```
Comment (juliasaveliff) [inline src/components/elementView/elementTable.tsx]: I don't know why git thinks I changed all the code around here (maybe automatic formatting changes???) but I don't think any of the code beyond this point actually changed

File: src/components/elementView/elementUtils.tsx [hunk 5]
```diff
@@ -12,6 +12,13 @@ export interface ITableSort {
   id: string;
 }
 
+export enum ElementTableOpenAlert {
+  NONE = "none",
+  DELETE = "delete",
+  GRANT_ADMIN = "grant_admin",
+  REVOKE_ADMIN = "revoke_admin"
+}
+
```
Comment (clairecfu) [inline src/components/elementView/elementUtils.tsx]: i LOVE THIS!!!!!

File: src/components/elementView/elementTable.tsx (hunk 2 above)
Comment (clairecfu) [reply src/components/elementView/elementTable.tsx]: trueeee

**Pull request merged**